pkg/globalnet/controllers/ipam/globalnet_ingress.go

```diff
@@ -0,0 +1,83 @@
+package ipam
+
+import (
+	"crypto/sha256"
+	"encoding/base32"
+	"fmt"
+	"strings"
+
+	"github.com/coreos/go-iptables/iptables"
+	"github.com/submariner-io/submariner/pkg/util"
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/klog"
+)
+
+func (i *Controller) initIPTableChains() error {
+	ipt, err := iptables.New()
```

Comment:
if you will call this function anyways, you might as well save the iptables object returned and avoid the initialization on all other places.

Reply:
after reading more code I see this is not the first creation of the iptables object. The comment still applies, just put it on the outermost function